src/sass/components/_about&gallery.scss

```diff
@@ -0,0 +1,163 @@
+
+@media screen and (max-width: 767px) {
+   
+    .how-its-made {
+      background-image: url(./images/mobile-about/background_milk.png),
+      url(./images/mobile-about/background_milk@2x.png);
+      background-size: 100%;
+      background-position-y: 47%;
+      background-position-x: center;
+      background-repeat: no-repeat;
+      padding-top: 125px;
+      padding-bottom: 100px;
+      margin: 0 auto;
+      background-color:#fff5f6;
+    
+    
+      &__label {
+      margin: 0;
+      margin-bottom: 15px;
+      font-family: 'Titan One', cursive;
+      font-weight: 400;
+      font-size: 18px;
+      line-height: 1.17;
+      text-align: center;
+      text-transform: uppercase;
+      color: $mainLabelColor;
+    
+       &__title {
+      margin: auto;
+      margin-bottom: 26px;
+      font-family: 'Titan One', cursive;
+      font-weight: 400;
+      font-size: 34px;
+      line-height: 1.15;
+      text-align: center;
+      letter-spacing: 0.04em;
+      text-transform: uppercase;
+      color: $mainTextColor;
+      max-width: 200px;
+    }
+}
+
+    
+    
+    .milk__btn {
+      display: inline-block;
+      margin-bottom: 39px;
+      padding-top: 13px;
+      padding-bottom: 13px;
+      padding-right: 37px;
+      padding-left: 37px;
+      min-width: 176px;
+      background: $secondTextColor;
+      border-radius: 22px;
+      border: 0;
+  
+      font-family: 'DM Sans', sans-serif;
+      font-weight: 700;
+      font-size: 14px;
+      line-height: 1.28;
+      color: $mainTextColor;
+      
+  
+      &__btn:hover,
```

Comment:
ten & da ci klasę ".milk__btn__btn:hover", a chyba chodziło o ".milk__btn:hover"

Reply:
to mogło dawać ci błąd w css